src/pages/drive/addIns/driveAddIns.tsx

```diff
@@ -0,0 +1,529 @@
+                    <td>Mobile device has internet access</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+                <tr>
+                    <td>state.deviceCommunicating</td>
+                    <td>Telematics device is communicating to the server</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+                <tr>
+                    <td>state.gpsConnected</td>
+                    <td>Mobile device has GPS enabled</td>
+                    <td>None</td>
+                    <td>Boolean</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>;
+
+
+const thirdPartyFromURISchema: ReactNode = <div className="paragraph">
+    <InformationalBox>
+        <p>Drive app v4.1.0+</p>
+    </InformationalBox>
+    <p>It's possible to open different applications like prontoforms or native calendar from add-ins. To do so, it's important to construct correct URI schema string and pass it to <code className="small-code-sample">window.open</code>. For example:</p>
+    <CodeSample
+        language="javascript"
+        code={`window.open(uriSchemaString, "_system")`}></CodeSample>
+    <p>Make sure to read carefully documentation of the app you're trying to open to use correct schema. For example, to open twitter application from addin you should use:</p>
+    <CodeSample
+        language="javascript"
+        code={`window.open("twitter://messages", "_system")`}></CodeSample>
+    <p>You can't use just <code className="small-code-sample">twitter://</code> as it's not correct and app won't open. You need to specify which page you want to open: <code className="small-code-sample">messages</code>, <code className="small-code-sample">account</code> etc.</p>
+    <p>To open webpage you need to use the same method, but with this notation:</p>
+    <CodeSample
+        language="javascript"
+        code={`window.open("https://google.com", "_blank")`}></CodeSample >
+    <InformationalBox>
+        <p><code className="small-code-sample">_blank</code> is important, especially for iOS devices</p>
+    </InformationalBox>
+</div>;
+
+
+const driveFromThirdParty: ReactNode = <div className="paragraph">
+    <p>On Android and iOS devices with the Geotab Drive app installed, a URL handler is registered which can:</p>
+    <ul>
+        <li>Launch Drive,</li>
+        <li>Navigate to a specified page in Drive, and</li>
+        <li>Automatically log into Drive with a token (session ID, user name, and database name) retrieved by authenticating against MyGeotab's API in a third-party app</li>
+    </ul>
+    <p>Deep linking is used to provide a seamless link from a third-party app into the Geotab Drive app.</p>
+    <p>On the most basic level, launching Geotab Drive to the main screen, can be executed by creating a link to:</p>
+    <CodeSample
+        language="javascript"
+        code={`geotabdrive://`}></CodeSample >
```

Comment:
```suggestion
        code={`geotabdrive://`}/>
```
This can be self closing cause it has no children inside of it.